src/models/videojs.js

```diff
@@ -109,7 +96,7 @@ const updatePosterUrl = (posterImage, width) => {
 
 class VideoJsPlayer extends Video {
 	constructor(el, opts) {
-		advertising = opts && opts['advertising'] ? true : false;
```

Comment:
why the opt name change? Will mean change across next aps, as in https://github.com/Financial-Times/next-front-page/blob/4a72955d75285184aed2570e1525d7e2b20a2b9b/client/main.js#L23?